authorization2/package.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "authorization",
+    "version": "1.0.0",
+    "description": "Authorization Typescript Node Lambda",
+    "main": "app.js",
+    "dependencies": {
+        "@aws-lambda-powertools/logger": "^1.5.0",
+        "@aws-lambda-powertools/metrics": "^1.5.0",
+        "@aws-sdk/client-dynamodb": "3.150.0",
+        "@aws-sdk/client-ssm": "^3.186.0",
+        "@aws-sdk/lib-dynamodb": "3.150.0",
+        "esbuild": "0.14.14",
+        "uuid": "^9.0.0"
```

Comment:
Is there a reason we're using uuid over native crypto/uuid 

- https://nodejs.org/api/crypto.html#:~:text=n%7D%60)%3B-,crypto.randomUUID(%5Boptions%5D),-%23

Reply:
Further reading: https://www.nearform.com/blog/new-crypto-capabilities-in-node-js/

Reply:
This has been amended to use crypto.randomUUID()